k8s/environments/ptl/cluster-00-overlay/kustomization.yaml

```diff
@@ -1,9 +1,3 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
```

Comment:
should this be an empty list ?

Reply:
Just what I did on ptlsbox, stops flux breaking, not that it matters as jenkins is the last app to be migrated.